FAQ.md

```diff
@@ -505,6 +505,31 @@ You can enable/disable lockdown mode in the main menu, using a widget or using a
 You can contribute translations [here](https://crowdin.com/project/netguard) (registration is free).
 If your language is missing, please contact me to have it added.
 
+<a name="FAQ54"></a>
+**(54) How to tunnel all TCP connections through the Tor network if you bought the SOCKS feature?**
+
+First, install [Orbot](market://details?id=org.torproject.android), the Android client for Tor,
+run it, press _Start_, while it connects open its _Settings_ and make sure it's setup to auto-start
+on device start.
+
+In NetGuard's _Advanced_ settings enable _Subnet routing_.
+
+Having purchased the NetGuard SOCKS feature (this is required), just enable it in _Advanced_
```

Comment:
Oh, saw it with a shopping basket at one time, indeed I did not see any info on this, so I still believed it was a $$$ feature.